packages/preview-middleware/README.md

```diff
@@ -98,12 +96,12 @@ server:
       rta:
         layer: CUSTOMER_BASE
         editors:
-          - path: /local/variant-editor.html
+          - path: /test/variant-editor.html
 ```
 
 
 ### Adaptation Project
-If you want to use the middleware in an adaption project, the additional `adp` object needs to be configured. This example would preview a local adaptation project merged with its reference application from the target system at `http://sap.example` and it will ignore certification validation errors. For adaptation projects, it is also recommended to add the `rta` configuration allowing to edit the project.
+If you want to use the middleware in an adaption project, the additional `adp` object needs to be configured. This example would preview a local adaptation project merged with its reference application from the target system at `http://sap.example` and it will ignore certification validation errors.
```

Comment:
Please revert the changes in this section as well

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1293/commits/33db76006778511bb50339eeb76fb9d8be910093